tests/functional/pages/surveys/household_composition/summary.page.js

```diff
@@ -0,0 +1,36 @@
+import SummaryPage from '../../summary.page'
+
+class HouseholdCompositionSummary extends SummaryPage {
+
+  clickEdit() {
+    browser.click('[data-qa="household-full-name-edit"]')
+    return this
+  }
+
+  isNameDisplayed(name) {
+    return browser.element('[data-qa="household-full-name-answer"]').getText() === name
+  }
+
+  areNamesDisplayed(names) {
```

Comment:
I think we are implementing complex logic here which is unnecessarily. We should use the chai testing framework to assert all values exist? i.e. return an array of names and assert those